### PR TITLE
Release Notes: python-dev required and 'python' variable deprecated

### PR DESCRIPTION
**Pull request by maxgalli**
…ecated

### CODE REVIEW

File: README/ReleaseNotes/v620/index.md [hunk 1]
```diff
@@ -364,6 +364,10 @@ prevent infinite loops or crashes.
   release.
 - The version of Python that ROOT has been built with is now exported in the
   CMake variable `${ROOT_PYTHON_VERSION}`, available after calling `find_package(ROOT)`.
+- python-dev is now required and the compilation will fail if CMake is not able
+  to find it.
+- CMake variable `-Dpython={OFF, ON}` is now deprecated in favour of `-Dpyroot={OFF,
```
Comment (etejedor) [inline README/ReleaseNotes/v620/index.md:369]: This bullet already exists (it is the first bullet in this section).

File: README/ReleaseNotes/v620/index.md [hunk 2]
```diff
@@ -364,6 +364,10 @@ prevent infinite loops or crashes.
   release.
 - The version of Python that ROOT has been built with is now exported in the
   CMake variable `${ROOT_PYTHON_VERSION}`, available after calling `find_package(ROOT)`.
+- python-dev is now required and the compilation will fail if CMake is not able
```
Comment (etejedor) [inline README/ReleaseNotes/v620/index.md:367]: Please place this bullet as the second in this section (right after the one that says that the `python` option is deprecated).

File: README/ReleaseNotes/v620/index.md [hunk 3]
```diff
@@ -441,6 +445,10 @@ shuts down the interpreter.
 `ctypes` types (`c_long`, `c_int`, `c_double`).
 - Added TPython.
 - Added support for `from ROOT import *` in Python2 (only).
+- python-dev is now required and the compilation will fail if CMake is not able
```
Comment (etejedor) [inline README/ReleaseNotes/v620/index.md:448]: I think the comment of @oshadura was not to repeat exactly in the Python section what you wrote in the Build section. @oshadura , can you tell @maxgalli exactly what you would like to have here?

Comment (oshadura) [reply README/ReleaseNotes/v620/index.md:448]: Let's ignore this part then, I am not sure what to say here.

**Comment by indexless** [comment]
Sorry, but I'm not sure python-dev is really **required**: https://root-forum.cern.ch/t/cannot-build-6-20-04-without-python-dev/38707/13  To verify this, try to install python-dev, then configure ROOT, switching **OFF** pyroot option, then uninstall python-dev, and finally 'make' it.

And it builds. At least for me -- I performed steps I've just enumerated, and it didn't fail.

-Timur

**Pull request closed without merging**